src/oasis-nexus/api.ts

```diff
@@ -777,3 +784,42 @@ export const useGetRuntimeEvmTokensAddressHolders: typeof generated.useGetRuntim
     },
   })
 }
+
+export const useGetConsensusAccounts: typeof generated.useGetConsensusAccounts = (
+  network,
+  params?,
+  options?,
+) => {
+  const ticker = getTickerForNetwork(network)
+  return generated.useGetConsensusAccounts(network, params, {
+    ...options,
+    request: {
+      ...options?.request,
+      transformResponse: [
+        ...arrayify(axios.defaults.transformResponse),
+        (data: generated.AccountList, headers, status) => {
+          if (status !== 200) return data
+          return {
+            ...data,
+            accounts: data.accounts.map(account => {
+              // TODO: remove when API returns total, looking at query filters we store that data in Nexus
+              const total = BigInt(account.available) + BigInt(account.escrow) + BigInt(account.debonding)
```

Comment:
This calculation is wrong. API does not return the correct fields, and debonding is named misleadingly. History of wrong math: https://github.com/oasisprotocol/nexus/issues/84 and https://github.com/oasisprotocol/oasis-wallet-web/pull/927

https://nexus.oasis.io/v1/consensus/accounts?limit=1
> "available":"0","debonding":"0","escrow":"0"

single account API returns correct fields
https://nexus.oasis.io/v1/consensus/accounts/oasis1qz78ap0456g2rk7j6rmtvasc9v2kjhz2s58qgj90
> "available":"21454060793","debonding":"0","debonding_delegations_balance":"0","delegations_balance":"103544825467","escrow":"0"

Reply:
I will leave TODOs in this case.